tests/file_storage/tests.py

```diff
@@ -290,6 +290,42 @@ def test_file_save_with_path(self):
 
         self.storage.delete('path/to/test.file')
 
+    def test_save_overwrite_behaviour(self):
+        """
+        Saving to same file name twice doesn't overwrite the first file
+        """
+        name = 'test.file'
+
+        self.assertFalse(self.storage.exists(name))
+
+        content_1 = b'content one'
+        content_2 = b'second content'
+        self.assertNotEqual(content_1, content_2)
+
+        f_1 = ContentFile(content_1)
+        f_2 = ContentFile(content_2)
+
+        stored_name_1 = self.storage.save(name, f_1)
+        self.assertEqual(stored_name_1, name)
+        self.assertTrue(self.storage.exists(stored_name_1))
+        self.assertTrue(
```

Comment:
Use single lines for all these asserts -- we allow up to 119 characters when it improves readability.